rules/nat_test.go

```diff
@@ -55,6 +56,37 @@ var _ = Describe("NAT", func() {
 			},
 		}))
 	})
+	It("should render rules when active with explicit port range", func() {
+
+		//copy struct
+		localConfig := rrConfigNormal
+		localConfig.NATPortRange = numorstring.Port{99, 100, ""}
```

Comment:
I think there's a utility function for making each type of port.  `numorstring.PortRange(a, b)`, maybe?